components/molecules/SearchMultiSelect.jsx

```diff
@@ -1,307 +1,376 @@
+                                    (option, i) =>
+                                        option.isSelected && (
+                                            <Tag
+                                                key={key}
+                                                className="-m-a-1"
+                                                context={context}
+                                                hasOption
+                                                onOptionClick={() => _removeOption(index, i)}
+                                                optionIcon="bt-times"
+                                                optionLabel="Remove option"
+                                                size={tagSize}
+                                                text={option.name}
+                                            />
+                                        )
+                                );
+                            }
+                            return;
+                        }
+                    })}
                 </div>
-                <div className="gds-search-select__menu">
-                    <div className={checkBoxClasses}>
-                        {matchingIndexes.map(index => {
-                            const { name, key, isSelected } = options[index];
-
+            </div>
+            <div className="gds-search-select__menu">
+                <div className={checkBoxClasses}>
+                    {matchingIndexes.length ? (
+                        matchingIndexes.map((option, i) => {
+                            const { name, value, isSelected, options: subOptions } = option;
                             const itemClass = cx('gds-search-select__menu-item', {
-                                'gds-search-select__menu-item--selected': index === currentIndex
+                                'gds-search-select__menu-item--selected': option === currentIndex,
                             });
-
-                            return (
-                                <div
-                                    key={key}
-                                    onClick={() => this._toggleOption(index)}
-                                    className={itemClass}>
-                                    <label className="gds-search-select__checkbox">
-                                        <input
-                                            name={name}
-                                            className="gds-search-select__checkbox-input"
-                                            type="checkbox"
-                                            checked={isSelected}
-                                            readOnly
+                            return !subOptions ? (
+                                <Checkbox
+                                    key={i}
+                                    value={value}
+                                    onChange={() => _toggleOption(option)}
```

Comment:
To select an option we have to click on the checkbox only, before it seems we could by clicking anywhere in the row. Is it possible to move back the `_toggleOption(option)` (and I think ` toggleSubItem(option, subIndex)` too) on the  parent div instead of the Checkbox ?

Reply:
good call .. thanks Morgane 👯 , its done!